shared/common-adapters/input.desktop.js

```diff
@@ -114,7 +114,6 @@ export default class Input extends Component {
           rows={this.props.rows}
           rowsMax={this.props.rowsMax}
           style={{...textStyle, ...globalStyles.flexBoxColumn, ...this.props.textStyle}}
-          textAlign='center'
```

Comment:
It appears that `textAlign` never existed as a prop on material-uis `TextField`, so I think it's very safe to remove. https://github.com/callemall/material-ui/commits/master/src/TextField/TextField.js